raiden_contracts/data_0.25.1/deployment_ropsten.json

```diff
@@ -0,0 +1 @@
+{"contracts_version": null, "chain_id": 3, "contracts": {"SecretRegistry": {"address": "0xA39faF9464b4E5668154003464e3a40ab9a83cB8", "transaction_hash": "0x948201d71da1c344ff722257e2153eafa2f7a55c1dd2afc72fbf61271d8bfa3f", "block_number": 5914539, "gas_cost": 298578, "constructor_arguments": []}, "TokenNetworkRegistry": {"address": "0x14375063E15B4cBe111c92aE35720d86FD8744bd", "transaction_hash": "0x8675a5b6889ff4766659a3e9fe899a02491522aa69129493aa9a0adf07929345", "block_number": 6553162, "gas_cost": 4139254, "constructor_arguments": ["0xA39faF9464b4E5668154003464e3a40ab9a83cB8", 3, 20, 555428, 115792089237316195423570985008687907853269984665640564039457584007913129639935]}}}
```

Comment:
should the version be `null`?

Reply:
This looks weird.  I'll try removing all these `null`s from the deployment files https://github.com/raiden-network/raiden-contracts/issues/1287 .